common/pinot/response_utility.go

```diff
@@ -83,12 +83,23 @@ func ConvertSearchResultToVisibilityRecord(hit []interface{}, columnNames []stri
 	if err != nil {
 		logger.Error("Unable to Unmarshal searchAttribute map", tag.Error(err))
 	}
+	var memo *p.DataBlob
+	if attributeMap["Memo"] != nil {
```

Comment:
Nit: do you mind making this key a const somewhere? Perhaps on this package as a publically accessible string?